docs/framework/20categories/20architecture/liquidity-networks.md

```diff
@@ -1,3 +1,24 @@
 ### Liquidity Networks
+Liquidity networks enable users to transfer already minted assets from one chain to another. With such protocols, there's no ‘bridging’ of assets, i.e., no assets move across chains. Instead, assets are exchanged between one user (liquidity providers, aka LPs) and another bridge user on different networks, resulting in a cross-chain swap. For example, a user receives the desired asset on the destination network via the LP. In return, the LP receives the user’s assets on the source network and fees for providing the desired asset. 
 
-Note: include HTLCs as a possible design approach
\ No newline at end of file
+Typically, LP funds within a liquidity network are collected and locked in a smart contract (liquidity pool) to facilitate quick cross-chain swaps. When a user initiates a transaction, the desired assets are released from the smart contract on the destination network. In essence, liquidity networks work similarly to peer-to-peer networks where LPs, acting as routers, own and distribute liquidity across chains. 
+
```

Comment:
I think it would be good to outline the key security properties we expect liquidity networks to guarantee here. We can then use these to analyze the approaches that are listed below.

Reply:
Based on the categorisations we've made, I'm not sure if all the key security properties would apply to all 3 categories since the risks (and the worst case scenario) in each type is different. For example, generally liquidity networks would have security properties like: 1) leveraging the security of the underlying blockchains - this is not true in cases of externally verified liquidity networks or 2) users only receive native assets - when liquidity is low, users can recieve hTokens for Hop and anyAssets for Multichain transfers.

Instead, like you've mentioned in the next comment, we can distinguish the categorisation based on liquidity networks that solely operate through exchange between LPs and users (atomic swaps) and partially/fully rely on the underlying messaging protocol and then mention these security features + risks in each section.

Let me know your thoughts!

Reply:
@ArjunChand13 I made some updates to this section in PR #43 , which is merged, so you can read it [here](https://crosschainriskframework.github.io/framework/20categories/20architecture/architecture/#liquidity-networks) as well. Can you kindly look at that and let me know what you think? It addresses some of the comments and makes some other write-up changes. 

Your second point above is a good one I hadn't thought of. I think we should add it to the considerations section.

Reply:
In the explanation of the off-chain mechanism, we've used "user then selects the LP" , "user identifies its preferred LP" - I'm not sure if the user has a choice here in terms of selecting the LP. Instead the user is matched with the LP with the best offer automatically by the liquidity network's mechanism.

Reply:
Yea, this process is largely transparent to the user and performed by an off-chain component/agent performing these steps on behalf of the user. I will rephrase it for clarity.

Reply:
Addressed in PR #60